docs/content/direct/get-started.md

```diff
@@ -16,6 +16,14 @@ This is one way to produce a very simple system, suitable for study but not prod
 
 ### Install software prerequisites
 
+If you're using WSL, it’s important to run the following commands with admin permissions to avoid permission-related issues. You can switch to root by running:
```

Comment:
Almost all of this file follows the reference "the following commands". Is that phrase intended to refer to all everything in this document, or only the rest of the `### Install software prerequisites` section? I recommend rewording for clarity.